pipeline.Dockerfile

```diff
@@ -36,11 +36,31 @@ RUN apt update -q \
     && apt autoremove -yqq --purge && apt clean && rm -rf /var/lib/apt/lists/* /var/log/*
 
 WORKDIR /kubespray
-COPY . .
 
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
     && pip install --no-compile --no-cache-dir pip -U \
-    && pip install --no-compile --no-cache-dir -r tests/requirements.txt -r requirements.txt \
+    && pip install --no-compile --no-cache-dir \
+        ansible==5.7.1 \
+        ansible-core==2.12.5 \
+        cryptography==3.4.8 \
+        jinja2==2.11.3 \
+        jmespath==0.9.5 \
+        MarkupSafe==1.1.1 \
+        netaddr==0.7.19 \
+        pbr==5.4.4 \
+        ruamel.yaml==0.16.10 \
+        ruamel.yaml.clib==0.2.7 \
+        ansible-lint==5.4.0 \
+        apache-libcloud==2.2.1 \
+        ara[server]==1.6.1 \
+        dopy==0.3.7 \
+        molecule==3.0.6 \
+        molecule-vagrant==0.3 \
+        python-vagrant==0.5.15 \
+        testinfra==5.2.2 \
+        tox==3.11.1 \
+        yamllint==1.19.0 \
+        kubernetes==26.1.0 \
     && KUBE_VERSION=$(sed -n 's/^kube_version: //p' roles/kubespray-defaults/defaults/main.yaml) \
```

Comment:
You probably would have noticed it when looking at the CI on first glance but yeah here is why the CI fail :p